homeassistant/components/camera/xiaomi.py

```diff
@@ -36,7 +35,7 @@
     vol.Required(CONF_MODEL): vol.Any(MODEL_YI,
                                       MODEL_XIAOFANG),
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.string,
-    vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
+    vol.Required(CONF_PATH): cv.string,
```

Comment:
This will break existing installations where the path matches. If a user uses a device with a different path then she/he can provide that path.